.github/CODEOWNERS

```diff
@@ -1,5 +1,4 @@
 # This is a CODEOWNERS file to indicate which people are responsible for this repository
 # These people will also be automatically requested for review on PRs
 
-* @waterloo-rocketry/default-reviewers @Darth-Raazi @BluCodeGH @KavinSatheeskumar
-
+* @waterloo-rocketry/default-reviewers @Darth-Raazi @BigLiban @patrick-gu @Lucifersan
```

Comment:
Add Saad in here too as he would to like to also be requested for review on PRs too